src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -655,25 +655,26 @@ public void TestParseTables_Negative(string script)
         }
 
         [Theory]
-        [InlineData("")]
-        [InlineData("a=10;")]
-        [InlineData("a=b=10;")]
-        [InlineData("a=10;c=20;")]
+        [InlineData(" ")]
+        [InlineData("a = 10;")]
+        [InlineData("   a    =  10    ;  ")]
+        [InlineData("a = b = 10;")]
+        [InlineData("a = 10; c = 20;")]
         public void TestFormulasParse(string script)
         {
             TestFormulasParseRoundtrip(script);
         }
 
         [Theory]
-        [InlineData("a=10")]
+        [InlineData("a = 10")]
         [InlineData("a;")]
         [InlineData(";")]
-        [InlineData("a=10;;")]
-        [InlineData("a=10;b")]
-        [InlineData("A=If(true,1;);")]
-        [InlineData("A=If(true,1;2);")]
-        [InlineData("1+2);")]
-        [InlineData("1=10")]
+        [InlineData("a = 10;;")]
+        [InlineData("a = 10; b")]
+        [InlineData("A = If(true,1;);")]
+        [InlineData("A = If(true,1;2);")]
+        [InlineData("1 + 2);")]
```

Comment:
what about multi character names?
Multi character names with special characters?
Ones with whitespace in the name, surrounded by `'`?

Reply:
Is there a way for me to know which special characters are allowed in identifiers?

Reply:
I updated the tests. I don't think any special characters are allowed in identifiers, but correct me if I'm wrong.

Reply:
Added tests with special characters inside of `'`